hyundai_kia_connect_api/Vehicle.py

```diff
@@ -262,12 +260,14 @@ def ev_estimated_station_charge_duration(self, value):
         self._ev_estimated_station_charge_duration = value[0]
 
     @property
-    def ev_charge_limits(self) -> EvChargeLimits:
-        return self._ev_charge_limits
+    def ev_charge_limits(self) -> int:
```

Comment:
It works, but if I were you, I'd just make the attributes `ev_charge_limits_ac` and `ev_charge_limits_ac` public (by removing the leading underscore) and remove the `ev_charge_limits` attribute. I don't think we need this extra level of complexity.

I'm using the term "public" loosely here, but it basically means the attributes can be read/written directly, without the need for getters and setters. It's the right way to do it in simple cases like this.

More on this here: https://towardsdatascience.com/whats-the-meaning-of-single-and-double-underscores-in-python-3d27d57d6bd1

I'm still getting familiar with the data model. It looks like using a dataclass for the Vehicle class was a good idea at the beginning, but the class has gotten bloated with getters and setters and I think some functions should be moved to the Vehicle class. But it's a breaking change, we can discuss later if it's worth doing.

Reply:
Makes sense. I will adjust to _ac and _dc attributes. I need to go through each API and align this too.